src/i18n/definitions.js

```diff
@@ -491,15 +493,18 @@ let strings = new LocalizedStrings(
       wordSourceDefaultText: "din tidligere læsning",
       wordSourcePrefix: "Ord fra",
 
-      //FindWordsInContext
+      //ExerciseHeadlines
       findTheWordInContextHeadline: "Find ordet/ordene i sammenhængen",
```

Comment:
Why the plural addition? In English it is only "the word".

Reply:
I've actually changed the English version as often it is more than one word :-)